spikes/stl_cxxsupport/main.cpp

```diff
@@ -94,5 +95,20 @@ auto main() -> int
 	auto back = span0.back();
 	log_info("span0.back() = %i", back);
 
+	log_info("Test std::range features");
+
+	constexpr auto range = [](const auto a, const auto b) {
+		auto x = std::ranges::views::iota(a) | std::ranges::views::take(b - a + 1);
+		return std::ranges::views::all(x);
+	};
```

Comment:
I suggest to move it to `utils` and ensure to use the same for spike and functional test

Reply:
can't do that yet as Apple's clang does not support std::ranges for now.
I'll look into using brew llvm's clang instead.